core/arch/opnd.h

```diff
@@ -2843,7 +2843,31 @@ DR_API
 bool
 reg_is_stolen(reg_id_t reg);
 
-/* internal version */
+/* Internal version */
+
+#ifdef X86
+
+/* Helper function to retrieve the "input size" as meant in Intel's Spec. Vol.2A "2.6.5
+ * Compressed Displacement (disp8*N) Support in EVEX". This is not a general function
+ * to determine the input size, and shall only be used for the purpose of table mentioned
+ * above.
+ */
+opnd_size_t
+instr_get_input_size_from_opcode(int opcode, bool evexw);
```

Comment:
I'm confused as to why this is in opnd.h: it's not operating on operands; its name doesn't start with opnd_ or reg_. 
 It seems like this should be in decode_private.h.  Ditto for both functions below.

Reply:
Moved, see above.